### DISCUSSION TITLE
Chromebook, Chrome OS

### DISCUSSION THREAD

**Discussion by baiti** [cgeo/cgeo#9865] [Ideas]
I had been using the very first Chromebooks in the US, around 2012 but gave up on it in the last recent years. Very recently I got a new Chromebook and noticed there is a "Play Store" on that thing. Sure enough I looked for cgeo und it said cgeo would not work on that device. Others, for instance Locus Map just works like a charm. So somehow this Chrome OS thing seems to be very much like an Android. I have not researched the details but let me just ask: What would it mean for cgeo to run on such a device? Is there a lot of extra work to perform or just enable some switch in a manifest here or there?

I noticed Locus Map runs on a small phone like screen on Chrome OS so it almost looks like an emulator, however I have a hard time to believe that it is indeed an emulator. But if it is not, why can't it run with full capabilities of the 1920x1080 screen?

So, ideally, if cgeo could be built such that it runs on Chrome OS, and run there in the device's display resolution, that would be extremely cool.

Anybody know more about the implications?

**Comment by baiti** [comment]
Let me add some relevant details: Why would I want to run cgeo on a Chromebook? Certainly  I wouldn't take it with me on an outdoor trip.

Now here is why it is relevant: A lot of preparatory work is done at home and using a Smartphone's capabilities for such work is a nightmare. It is much more convenient to use a Laptop and perhaps just export a GPX to Google Drive and pull it into cgeo running on the phone from there. Also, in the backend, I might want to use cgeo on the Chromebook to type up my field notes logs and then submit them to the platform of my choice.

**Comment by eddiemuc** [comment]
https://developer.android.com/topic/arc

**Reply by baiti** [reply]
thx for the pointer @eddiemuc ... interesting stuff
